llvm/lib/IR/DebugInfo.cpp

```diff
@@ -80,6 +80,23 @@ TinyPtrVector<DbgVariableRecord *> llvm::findDVRDeclares(Value *V) {
   return Declares;
 }
 
+TinyPtrVector<DbgVariableRecord *> llvm::findDVRValues(Value *V) {
+  // This function is hot. Check whether the value has any metadata to avoid a
+  // DenseMap lookup.
+  if (!V->isUsedByMetadata())
```

Comment:
How expensive is this check then? Is that a field lookup, a DenseMap lookup, ...? Might be worth commenting that here.